special-interest-groups/observability-tag/charter.md

```diff
@@ -0,0 +1,48 @@
+# OpenSearch Observability Technical Advisory Group (TAG) Governance Charter
+
+### Purpose
+
+The OpenSearch Project proposes to create a Observability Interest Group that focuses on observability using OpenSearch. This includes log analytics, traces, metrics using OpenSearch and related projects such as Prometheus, OpenTelemetry. This group will also establish guidelines for members, set architecture guidelines, and come up with recommendations which will then be reviewed by TSC for decision making. The group will provide feedback to the TSC before the final decision is made through voting of TSC members.
+
+### Responsibilities
+
+The following is a non-exhaustive sample list of activities and deliverables that are in-scope for this SIG
+
+* Drive feature development and enhancements for observability applications using OpenSearch (e.g. ingestion, plugins, Alerting, Anomaly Detection, Dashboards).
+* Align OpenSearch observability initiatives with industry trends, CNCF observability guidelines, and interoperability standards.
+* Collaborate with external observability projects like OpenTelemetry, Prometheus, Jaeger, and Fluent Bit to enable seamless data ingestion, integration and interoperability. 
+* Define and promote observability best practices for OpenSearch deployments, covering data ingestion pipelines, alerting, visualization, and operational monitoring.
+* Coordinate with other OpenSearch SIGs and CNCF Operational Resilience TAG to align roadmaps, share insights, and contribute to cross-community initiatives.
```

Comment:
The CNCF TAG is called "TAG operational Resilience". The charter of that group is not to create a roadmap. see https://docs.google.com/document/d/1bDAIevQ_69bV1gnpDRgrxNubhyazf8flRWYgkKAf5jA/edit?tab=t.0#heading=h.z7nikj46tsbw